web/content/index.adoc.erb

```diff
@@ -16,9 +16,6 @@ For Foreman plugin overview, see link:https://theforeman.org/plugins/[Plugins].
 
 Start by using the top menu bar to select your release version.
```

Comment:
Please take this as future thoughts.

Building on what you started below, I would love to shorten this further. For example, this whole section could also be dropped IMHO. My mind went to expanding the releases we list (both under `Supported releases` and `Pre-release versions`) be link to the specific flavors.

Just a proposal:

* Foreman 3.15 (link to `/release/3.15`)
  * On Enterprise Linux (link to `/release/3.15/index-foreman-el.html`)
  * On Debian (link to `/release/3.15/index-foreman-deb.html`)
  * With Katello 4.17 (link to `/release/3.15/index-katello.html`)

Then you don't need to list the available flavors and people immediate go to the overview they want.